### DISCUSSION TITLE
504 Gateway Time-out on both front-end and admin

### DISCUSSION THREAD

**Discussion by dolphiq-allard** [craftcms/cms#15896] [Q&A]
Hey guys,

I've been getting 504 Gateway Time-outs  in Craft CMS lately. At first I thought it had to do with my hosting party (Cloudways) but now I also have it on my development machine (Docker). I've dropped all tables, removed my complete project config and installed Craft clean from there on. However, now after adding a few fields, it just stops working.

I suspect there's something going on in the DB making it stop.

Below is an overview of what CloudWays saw on the production site slowing things down.

![image](https://github.com/user-attachments/assets/3a963d75-1e32-42ed-ab22-7173088420da)
 
After backing up the database, restoring it with dropping all tables everything works fine for a couple of minutes before again taking forever to load or ultimately resulting in a 504.

Really don't know where to start looking. Any help would be greatly appreciated.

Cheers

**Comment by brandonkelly** [comment]
What database/version are you using?

**Reply by dolphiq-allard** [reply]
Using MySQL. However, no need to dive into this, I deleted the project and started it from scratch using DDEV and all seem to work fine now. Think there was something wrong with my DB.

Thanks.
